subprojects/packagefiles/lodepng/meson.build

```diff
@@ -0,0 +1,18 @@
+project('lodepng', ['c', 'cpp'],
+    version: '20210627',
+    license: 'zlib')
+
+# C++ version
+lodepng_dep = declare_dependency(
+    include_directories: include_directories('.'),
+    sources: files('lodepng.cpp'))
+
+# C89 version
+copy = find_program('cp', 'copy', required: true)
+run_command(copy, 'lodepng.cpp', 'lodepng.c')
```

Comment:
This is writing to the source tree...

Reply:
Use configure_file(..., copy: true) to copy and/or rename a file into the build directory instead.

Reply:
> Use configure_file(..., copy: true) to copy and/or rename a file into the build directory instead.

Thank you.